app/components/support_interface/personal_details_component.rb

```diff
@@ -90,6 +90,8 @@ def date_of_birth_row
       {
         key: 'Date of birth',
         value: application_form.date_of_birth ? application_form.date_of_birth.to_s(:govuk_date) : MISSING,
+        action: 'Change',
```

Comment:
All of these change links need to include visually hidden text too. So when read aloud this says ‘Change date of birth’. Think there should be examples of how to do this elsewhere in the code base.

Reply:
Hey Paul, would appreciate if you could show me what you mean by this? had a quick look through the app and not 100% clear what you mean.

Reply:
If you look in `SummaryListComponent`, you’ll see that by providing `change_path` instead of `action_path`, then the value for `action` gets shown in the hidden text, and ‘Change’ is shown by default.

https://github.com/DFE-Digital/apply-for-teacher-training/blob/06b0970b6f6157ddd55314cc2d930d08a75547ef/app/components/summary_list_component.html.erb#L19-L25

Here’s an example of this component being called from another view in the app:

```rb
def name_row(reference)
  {
    key: 'Name',
    value: reference.name,
    action: "name for #{reference.name}",
    change_path: candidate_interface_references_edit_name_path(reference.id),
  }
end
```

Reply:
Okay have done so, ready for a re-review